axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceLineController.java

```diff
@@ -470,7 +470,12 @@ public void setAxisDomains(ActionRequest request, ActionResponse response)
   public void manageInvoiceLineAxis(ActionRequest request, ActionResponse response)
       throws AxelorException {
     try {
-      Invoice invoice = request.getContext().getParent().asType(Invoice.class);
+      InvoiceLine invoiceLine = request.getContext().asType(InvoiceLine.class);
+      Invoice invoice = invoiceLine.getInvoice();
+      Context parentContext = request.getContext().getParent();
+      if (parentContext != null && Invoice.class.equals(parentContext.getContextClass())) {
+        invoice = parentContext.asType(Invoice.class);
+      }
```

Comment:
There is an already method to do this : 

https://github.com/axelor/axelor-open-suite/blob/2db2f4e055beb446631271da610057499e93da04/axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceLineController.java#L314